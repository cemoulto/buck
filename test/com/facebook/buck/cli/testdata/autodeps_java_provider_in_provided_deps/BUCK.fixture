java_library(
  name = 'foo',
  srcs = [
    'Foo.java',
  ],
  autodeps = True,
  provided_deps = [
    ':baz',
  ],
)

java_library(
  name = 'bar',
  srcs = [
    'Bar.java',
  ],
  autodeps = True,
)

java_library(
  name = 'baz',
  srcs = [
    'Baz.java',
  ],
  autodeps = True,
)
