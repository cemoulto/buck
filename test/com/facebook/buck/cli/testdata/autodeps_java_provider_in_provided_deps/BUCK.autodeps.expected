#@# GENERATED FILE: DO NOT MODIFY 31d4b2f2ff5619a5d967b3186fc42c9d6888dd96 #@#
{
  "foo" : {
    "deps" : [
      "//:bar"
    ],
    "exported_deps" : [ ]
  }
}
